index.js

```diff
@@ -662,16 +662,19 @@ class GithubScm extends Scm {
     async _getCommitRefSha(config) {
         try {
             const commit = await this.breaker.runCommand({
-                action: 'getCommitRefSha',
+                action: 'getCommit',
```

Comment:
Action name has changed, but we didn't fix the function name. The role of this function is same, it still returns commit ref sha. Is this OK?